final_prep/annotations.rb

```diff
@@ -3,41 +3,63 @@
 # Use the # to create a new comment
 
 # Build a Bear
-
+# Defines a method called build_a_bear, allows for arguments, name, age, fur, clothes, and specail_power.
 def build_a_bear(name, age, fur, clothes, special_power)
+  #defines varable greeting with string output including bears name,
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  #defines varabile demographics as an array including the bears name and age
   demographics = [name, age]
+  #defines the variable power_saying to a string that interoplates the special_power variable
   power_saying = "Did you know that I can #{special_power}?"
+  # sets a hash called built_bear, with key/value pairs for basic info, clothes, exterior, cost, sayings, and a boolean for is_cuddly
   built_bear = {
+    #key value pair, key = basic_info, value is demographics varible
     'basic_info' => demographics,
+    #key value pair for clothes as key, and clothes variable as value
     'clothes' => clothes,
+    #key value pair, sets value to "exterior" and value to fur variable
     'exterior' => fur,
+    #key value pair, sets key to cost, and value to float 49.99
     'cost' => 49.99,
+    #key value pair, sets key "sayings" to a value of an array
     'sayings' => [greeting, power_saying, "Goodnight my friend!"],
+    #sets key value is_cuddly to a boolean of true
     'is_cuddly' => true,
   }
+  #returns the key value pairs for the built_bear hash after arguments have been passed threw it
   return built_bear
+  #end of method
 end
-
+# calls the method with arguements, using strings, intigers and arrays
 build_a_bear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares')
 build_a_bear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in')
 
 
 # FizzBuzz
-
+# defines method called fizzbuzz, allows for 3 arguments, num_1, num2, and range
```

Comment:
These are called _parameters_. Arguments are the values that you actually pass into the method (i.e. on lines 63 and 65). Parameters and arguments are 2 sides of the same coin, so to speak.